### DISCUSSION TITLE
Unclosed client session

### DISCUSSION THREAD

**Discussion by lunar54** [Chia-Network/chia-blockchain#11854] [Support]
Fresh Install of Ubuntu 20.04 Desktop Full Fresh Sync of wallet. wallet version 1.3.5 installed via apt 
After plot copy's to Final Drive in log finding 
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7f2f0a52cd60>

**Comment by loppefaaret** [comment]
are you using the multithreaded madmax plotter ?

**Reply by lunar54** [reply]
yes
